src/main/flight/pid.c

```diff
@@ -266,6 +267,10 @@ float pidGetFfSmoothFactor()
 {
     return pidRuntime.ffSmoothFactor;
 }
+float pidGetFfJitterFactor()
```

Comment:
```suggestion
float pidGetFfJitterFactor() {
```

Reply:
Only some cosmetics - we are using both it seems but as I'm old school I prefer one line.

Reply:
Yes maybe I can do this later, once we get the PR into master, since more work is needed in these places anyway.